### PR TITLE
New functions to libdraw

### PR DESCRIPTION
**Pull request by fuchicar**
+ Now we can draw 3D borders
 + We can change default menu colors.

Signed-off-by: fuchicar <rafita.fernandez@gmail.com>

### CODE REVIEW

File: sys/src/libdraw/menuhit.c [hunk 1]
```diff
@@ -32,29 +32,55 @@ static	Image	*bord;
 static	Image	*text;
 static	Image	*htext;
 
+static void
+setimagemenucolors(Image	*backcolor, Image	*highcolor, Image	*bordercolor, Image	*textcolor, Image	*seltextcolor){
```
Comment (gmacd) [inline sys/src/libdraw/menuhit.c]: Have some tabs sneaked in here?

**Comment by gmacd** [comment]
Any screenshots? :)

**Comment by elbing** [comment]
There are part of the upcoming PR...

File: sys/src/libdraw/menuhit.c (hunk 1 above)
Comment (fuchicar) [reply sys/src/libdraw/menuhit.c]: mmm, let me fix it. It's my editor.

**Pull request merged**